### PR TITLE
Container-Runtime and K8S should not be in the management scope of keadm.

### PR DESCRIPTION
**Pull request by bitvijays**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
/kind feature

**What this PR does / why we need it**:
What Changed?

- Previously, during keadm init keadm checks whether Kubernetes and Docker is installed or not. If not installed, it installs them.
- Now, keadm only checks if Kubernetes is installed or not. If not, throws an error and request user to install Kubernetes.
- Commented code for
 - Check and installation of Docker.
 - Installation of Kubernetes, instead display a error message.

Why this change was made?

- Issue 1117
- Container-Runtime and K8S should not be in the management scope of keadm #1117
**Special notes for your reviewer**:
Please ignore the multiple commit history. Still newbie to the git.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by bitvijays** [comment]
@fisherxu Have done the initial changes to fix Issue 1117. Let me know if initial changes are alright. Would be happy to do the changes.

Let me know your thoughts.

File: keadm/app/cmd/util/k8sinstaller.go [hunk 1]
```diff
@@ -55,14 +55,10 @@ func (ks *K8SInstTool) InstallTools() error {
 		ks.SetK8SVersionAndIsNodeFlag(ks.DefaultToolVer, ks.IsEdgeNode)
 		fallthrough
 	case types.NewInstallRequired:
-		err := ks.InstallK8S()
-		if err != nil {
-			return err
-		}
+		return fmt.Errorf("Kubernetes %s is not installed! Kindly install Kubernetes", component)
```
Comment (subpathdev) [inline keadm/app/cmd/util/k8sinstaller.go]: Add 'https://kubernetes.io/de/docs/setup/' to the output please.

File: keadm/app/cmd/cloud/init.go [hunk 2]
```diff
@@ -29,21 +29,16 @@ import (
...
+It checks if the pre-requisites are installed already.
 `
 	cloudInitExample = `
 keadm init
 
-- This command will download and install the default version of pre-requisites and KubeEdge
+- This command will download and install the default version of KubeEdge
 
-keadm init --kubeedge-version=0.2.1 --kubernetes-version=1.14.1 --docker-version=18.06.3 --kube-config=/root/.kube/config
+keadm init --kubeedge-version=0.2.1 --kube-config=/root/.kube/config
```
Comment (subpathdev) [inline keadm/app/cmd/cloud/init.go]: please update the kubeedge version to 1.1.0

**Comment by fisherxu** [comment]
Duplicate with #1475, close it first :)

**Pull request closed without merging**